content/tutorials/quickstart/py-create-time-based-escrows.md

```diff
@@ -0,0 +1,639 @@
+
+## Create Escrow
+
+<div align="center">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/L_2sKokW5E4?si=6r9vn4ojr6b42H2E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+You can create a time-based escrow with a minimum time to finish the escrow and a cancel time after which the funds in escrow are no longer available to the recipient. This is a test harness: while a practical scenario might express time in days or weeks, this form lets you set the finish and cancel times in seconds so that you can quickly run through a variety of scenarios. (There are 86,400 seconds in a day, if you want to play with longer term escrows.)
+
+To create a time-based escrow:
+
+1. Enter an **Amount** to transfer. For example, _100000000_.
+2. Copy the **Operational Account** value.
+3. Paste it in the **Destination Account** field.
+4. Set the **Escrow Finish (seconds)** value. For example, enter _10_.
+5. Set the **Escrow Cancel (seconds)** value. For example, enter _120_.
+6. Click **Create Time-based Escrow**.
+7. Copy the _Sequence Number_ of the escrow called out in the **Standby Result** field.
+
+The escrow is created on the XRP Ledger instance, reserving 100 XRP plus the transaction cost. When you create an escrow, capture and save the **Sequence Number** so that you can use it to finish the escrow transaction.
+
+[![Completed Create Escrow Transaction](img/quickstart-py-escrow3.png)](img/quickstart-py-escrow3.png)
+
+## Finish Escrow
+
+The recipient of the XRP held in escrow can finish the transaction any time within the time window after the Escrow Finish date and time but before the Escrow Cancel date and time. Following on the example above, you can use the _Sequence Number_ to finish the transaction once 10 seconds have passed.
+
+To finish a time-based escrow:
+
+1. Paste the sequence number in the Operational account **Sequence Number** field.
+2. Click **Finish Escrow**.
+3. Click **Get Op Account Info** and **Get Standby Account Info** to update their **XRP Balance**.
+
+The transaction completes and balances are updated for both the Standby and Operational accounts.
+
+[![Completed Escrow Transaction](img/quickstart-py-escrow4.png)](img/quickstart-py-escrow4.png)
+
+## Get Escrows
+
+Click **Get Escrows** to see the current list of escrows for the Operational account. If you click the buttons now, there are no escrows at the moment.
+
+For the purposes of this tutorial, follow the steps in [Create Escrow](#create-escrow), above, to create a new escrow transaction. Remember to capture the _Sequence Number_ from the transaction results.
+
+Click **Get Escrows**. The `account_info` request returns the same `account_object` for both accounts, demonstrating the link between the accounts created by the escrow transaction.
```

Comment:
I think you meant `account_objects` instead of `account_info`

Reply:
Added a nod to the reserve requirements in the paragraph following the create instructions.

You're right about account_info/account_object, but that paragraph is actually a copy/paste error, because the JavaScript example is different than the Python version. I removed that paragraph.